docs/components/topnav-primary/test.html

```diff
@@ -0,0 +1,71 @@
+---
+title: Testing - Top Navigation
+---
+{% extends 'no-frame.njk' %}
+
+{% block content %}
+<div id="vue-eyebrowDemo">
+  <div id="hxTopnav-primary">
+    <a class="img-left brand-logo" href="#"> <img src="images/helix-logo.svg" alt="Logo" /></a>
+    <div id="right-menu">
+      <div>
+        <a href="#" @click="selectedTag(tab)" :current-tab="currentTab">
+          <hx-icon type="bell"></hx-icon>
+          <p>Notifications</p>
+        </a>
+
+        <a href="#" class="selected">
+          <hx-icon type="ticketing"></hx-icon>
+          <p>Tickets</p>
+        </a>
+
+        <a href="#">
+          <hx-icon type="support"></hx-icon>
+          <p>Support</p>
+        </a>
+
+        <a href="#" class="disabled">
+          <hx-icon type="billing"></hx-icon>
+          <p>Billing</p>
+          <hx-tooltip  position="top-center">
+            Top Center Tooltip
+          </hx-tooltip>
+        </a>
+
+        <a href="#">
+          <hx-icon type="account"></hx-icon>
+          <p>Account</p>
+        </a>
+      </div>
+      <div class="straightLine"></div>
+      <div>
+        <hx-disclosure id="topnav-menu" aria-controls="eyebrowMenuId" role="button" aria-expanded="true" class="disabled">
+          <span>Drop-down label</span>
+          <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
+        </hx-disclosure>
+        <hx-menu id="eyebrowMenuId">
+          <div id="sectionOne">
+            <hx-menuitem role="menuitem" id="menuKey">Account Number :</hx-menuitem>
+            <hx-menuitem role="menuitem" id="menuValue">12345678</hx-menuitem>
+          </div>
+          <hr class="divider">
+          <div id="sectionTwo">
+            <header id="sectionTitle">TITLE</header>
+            <hx-menuitem role="menuitem">Action 1</hx-menuitem>
+            <hx-menuitem role="menuitem" class="disabled">Action disabled</hx-menuitem>
+            <hx-menuitem role="menuitem">Action 3</hx-menuitem>
+          </div>
+          <hr class="divider">
+          <div id="sectionThree">
+            <button class="hxBtn">Log Out </button>
+          </div>
+        </hx-menu>
+      </div>
+    </div>
+  </div>
+
+  <footer>
+    <pre><code v-text="snippet"></code></pre>
```

Comment:
You can remove the snippet from the Test Page.

Reply:
Changes have been done, please review.